ui/page/dcrdex/dcrdex_page.go

```diff
@@ -34,6 +39,9 @@ func NewDEXPage(l *load.Load) *DEXPage {
 		MasterPage:        app.NewMasterPage(DCRDEXID),
 		openTradeMainPage: l.Theme.NewClickable(false),
 	}
+
+	dp.initSplashPageWidgets()
```

Comment:
This method only does 2 things. Doesn't need to be a method.

Reply:
Method is used by trade page to display DCRDEX page